WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -2017,9 +2017,14 @@ - (BOOL)assetsPickerController:(CTAssetsPickerController *)picker shouldSelectAs
                                 message:NSLocalizedString(@"This image is currently not available locally to be used in your post. Please try again later.", @"User information explaining that the image is not available locally. This is normally related to share photo stream images.")];
```

Comment:
as suggested in [previous pr](https://github.com/wordpress-mobile/WordPress-iOS/pull/3028#discussion_r22801583), pls standardize to: 

@"This Photo Stream image cannot be added to your WordPress. Try saving to your Camera Roll before uploading."

> OK